charts/kcp/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: kcp
 description: A prototype of a multi-tenant Kubernetes control plane for workloads on many clusters
 
 # version information
-version: 0.2.5
-appVersion: "0.10"
+version: 0.3.0
+appVersion: "0.30"
```

Comment:
can we maybe make this `0.20` for the current kcp release? I know we use the `latest` tag, but I would like to reference an app version that exists at least.